build/release.sh

```diff
@@ -27,6 +23,32 @@ reset_to_master() {
 
     # Fetch latest code with tags
     git fetch --tags github-upstream || return 1;
+}
+
+
+reset_to_previous_version() {
+    if OLD_VERSION === "XXX"; then
+        echo "----------------------------------------------------------------------"
+        echo "Error while cleaning workspace!"
+        echo "----------------------------------------------------------------------"
+        return 1;
+    fi
+
+    # Reset and fetch upstream with tags
+    reset_tags || return 1;
```

Comment:
If you end up staying with this approach, I would add a message saying what you did